modules/pam_access/pam_access.c

```diff
@@ -34,6 +34,7 @@
 #include <stdarg.h>
 #include <syslog.h>
 #include <string.h>
+#include <libgen.h>
```

Comment:
Please don't include <libgen.h> and don't use functions declared there unless you really have to.